src/Datasource/Paging/NumericPaginator.php

```diff
@@ -578,7 +578,7 @@ protected function validateSort(RepositoryInterface $object, array $options): ar
     /**
      * Remove alias if needed.
      *
-     * @param array<string, mixed> $fields Current fields
+     * @param array<string|int, mixed> $fields Current fields
```

Comment:
This methods wants string keys only.

Reply:
Well yes we expect it to only receive it but it could be, that users are passing on simple arrays instead of associative ones. Thats why we have the `if (is_int($field))` check in there.
To make stan happy we have to tell it that it could be a simple array as well

Reply:
Then I'll close this PR because we have to then overwrite the native type hints with what we actually expect from the user with the phpdoc typehints. Guess this can't be fixed natively if PHP doesn't provide granular array types.